contrib/proxy/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  proxy:
+    image: gopackages/proxy
```

Comment:
how come our github's org is `gomods` but our docker's org is `gopackages`? 😄

Reply:
I don't think it is :-) https://hub.docker.com/r/gomods/

Reply:
that's the old one

Reply:
My bad - I'll fix this 👍